packages/service-metrics/package.json

```diff
@@ -43,6 +43,7 @@
     "@jest/globals": "^29.5.0",
     "@opentelemetry/api": "^1.3.0",
     "@opentelemetry/exporter-metrics-otlp-http": "^0.34.0",
+    "@opentelemetry/exporter-prometheus": "^0.40.0",
```

Comment:
@nathanielc i think this might have incompatible dependencies with the other stuffs is why js-ceramic might have broke but i am not sure